pkg/cli/admin/release/mirror.go

```diff
@@ -442,5 +451,62 @@ func (o *MirrorOptions) Run() error {
 	} else {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
+
+	printImageContentInstructions(o.Out, o.From, o.To, repositories)
+	return nil
+}
+
+// printImageContentInstructions provides exapmles to the user for using the new repository mirror
+// https://github.com/openshift/installer/blob/master/docs/dev/alternative_release_image_sources.md
+func printImageContentInstructions(out io.Writer, from, to string, repositories map[string]struct{}) error {
+	type installConfigSubsection struct {
+		ImageContentSources []operatorv1alpha1.RepositoryDigestMirrors `json:"imageContentSources"`
+	}
+
+	var sources []operatorv1alpha1.RepositoryDigestMirrors
+
+	mirrorRef, err := imagereference.Parse(to)
+	if err != nil {
+		return fmt.Errorf("Unable to parse image reference '%s': %v", to, err)
+	}
+	mirrorRepo := mirrorRef.AsRepository().String()
+
+	sourceRef, err := imagereference.Parse(from)
+	if err != nil {
+		return fmt.Errorf("Unable to parse image reference '%s': %v", from, err)
+	}
+	sourceRepo := sourceRef.AsRepository().String()
+	repositories[sourceRepo] = struct{}{}
+
+	for repository := range repositories {
+		sources = append(sources, operatorv1alpha1.RepositoryDigestMirrors{
+			Source:  repository,
+			Mirrors: []string{mirrorRepo},
+		})
+	}
+
+	// Create and display install-config.yaml example
+	imageContentSources := installConfigSubsection{
+		ImageContentSources: sources}
+	installConfigExample, err := yaml.Marshal(imageContentSources)
+	if err != nil {
+		return fmt.Errorf("Unable to marshal install-config.yaml example yaml: %v", err)
+	}
+	fmt.Fprintf(out, "\nTo use the new mirrored repository to install, add the following section to the install-config.yaml:\n")
+	fmt.Fprintf(out, string(installConfigExample))
+
+	// Create and display ImageContentSourcePolicy example
+	icsp := operatorv1alpha1.ImageContentSourcePolicy{
+		Spec: operatorv1alpha1.ImageContentSourcePolicySpec{
+			RepositoryDigestMirrors: sources,
+		},
+	}
+	icspExample, err := yaml.Marshal(icsp)
```

Comment:
This unfortunately creates creationTimestamp nil.  You might want to convert to unstructured, then remove the creationTimestamp field, then marshal to yaml.